roles/coral_credits/defaults/main.yaml

```diff
@@ -0,0 +1,101 @@
+---
+
+# The chart to use
+coral_credits_chart_repo: https://stackhpc.github.io/coral-credits
+coral_credits_chart_name: coral-credits
+coral_credits_chart_version: 0.3.5
+
+# Release information for the coral credits release
+coral_credits_release_namespace: "{{ azimuth_release_namespace | default('azimuth') }}"
```

Comment:
Can we put coral-credits in its own namespace by default please?

Reply:
Set default to coral-credits